recipe/meta.yaml

```diff
@@ -1,32 +1,36 @@
 {% set name = "SQLAlchemy-JSONField" %}
-{% set version = "1.0.0" %}
+{% set version = "1.0.1.post0" %}
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: 766d0b25bdebf53f67ccfaf9975987f921965987b37bae3a95ba6e7855afe98b
+  sha256: 72a5e714fe0493d2660abd7484a9fd9f492f493a0856288dd22a5decb29f5dc4
 
 build:
   number: 0
-  script: {{ PYTHON }} -m pip install . -vv
-  noarch: python
+  skip: True  # [py<37]
+  script: {{ PYTHON }} -m pip install . --no-deps --no-build-isolation -vv
 
 requirements:
   host:
-    - python >=3.5
+    - python
     - pip
     - setuptools
     - setuptools_scm
+    - wheel
   run:
-    - python >=3.5
+    - python
     - sqlalchemy
-    - typing >=3.6
 
 test:
   imports:
     - sqlalchemy_jsonfield
+  requires: 
+    - pip
+  commands: 
+    - pip check
```

Comment:
Do we want to add functional tests or we can skip those?

Reply:
I've added a test suite